src/integration/common/radioSchedule.js

```diff
@@ -1,7 +1,7 @@
 const servicesWithRadioSchedule = ['indonesia'];
 const servicesWithoutRadioSchedule = ['gahuza'];
 
-export default ({ isAmp }) => {
+export default ({ isAmp = false }) => {
```

Comment:
I think we should revert this change - I believe this test is called from other pages which may or may not have a radio schedule, and defaulting the value to false might still be required.

If it's not used from anywhere, let's remove the `isAmp` param and all references altogether.

Reply:
I will remove the default false value, but leave isAmp as your belief is correct and this test is also used in the live radio pages. I can remove it in the live radio PR if we merge them in that order.